docs/supported-technologies.asciidoc

```diff
@@ -89,6 +89,21 @@ be supported.
 
 The Python APM agent comes with automatic instrumentation of various 3rd party modules and standard library modules.
 
+[float]
+[[automatic-instrumentation-scheduling]]
+=== Scheduling
+
+[float]
+[[automatic-instrumentation-scheduling-celery]]
+===== Celery
+
+We support these Celery versions:
+
+* 3.x
+* 4.x
+
+Celery tasks will be recorded with Django and Flask only.
```

Comment:
```suggestion
Celery tasks will be recorded automatically with Django and Flask only.
```

Also, it might make sense to document how the instrumentation can be hooked up manually, in a separate PR of course.